drivers/hmis/app/graphql/types/hmis_schema/referral_posting.rb

```diff
@@ -0,0 +1,80 @@
+###
+# Copyright 2016 - 2023 Green River Data Analysis, LLC
+#
+# License detail: https://github.com/greenriver/hmis-warehouse/blob/production/LICENSE.md
+###
+
+module Types
+  class HmisSchema::ReferralPosting < Types::BaseObject
+    field :id, ID, null: false
+
+    # Fields that come from Referral
+    field :referral_identifier, ID, null: true
+    field :referral_date, GraphQL::Types::ISO8601Date, null: false
+    field :referred_by, String, null: false
+    field :referral_notes, String, null: true
+    field :chronic, Boolean, null: true
+    field :score, Integer, null: true
+    field :needs_wheelchair_accessible_unit, Boolean, null: true
+
+    # Fields that come from ReferralHouseholdMembers
+    field :hoh_name, String, null: false
+    field :household_size, Integer, null: false
+
+    # Fields that come from Posting
+    field :resource_coordinator_notes, String, null: true
+    field :posting_identifier, ID, null: false, method: :identifier
```

Comment:
This will need to be nullable, to support postings that are generated from within the HMIS. (The flow where you "Refer out" from a different project)

Reply:
thanks, I missed that one